blurb/blurb.py

```diff
@@ -749,6 +749,10 @@ def find_editor():
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+    editor = subprocess.check_output(
+        ["git", "config", "--get", "core.editor"]).strip()
+    if editor:
+        return editor
```

Comment:
If it has a default value of "vi" *anywhere* then I really don't want to use it.  vi is a fine editor--I've been using it for thirty years--but it's confusing for beginners, and beginners are exactly the population who won't have an explicit editor set.  I already put my foot down about explicitly listing "vi" as a default editor in blurb.